Containerfile.rag

```diff
@@ -1,21 +1,17 @@
 # vim: set filetype=dockerfile
-
-# port, commands, and other details inherited from the base image
 ARG OLS_VERSION=latest
+ARG OCP_VERSION=4.15
+
+FROM quay.io/openshift/lightspeed-rag-content:${OLS_VERSION} as lightspeed-rag-content
```

Comment:
i think you're going to need to stage this in multiple PRs because you first need to introduce the PR that produces the rag-content image, and we need to build+publish that.

then you can introduce the PR that pivots the OLS image to grab the RAG content from the image instead of from the zip file.

while technically the way our builds work, the OLS image won't use the quay.io image, it will use the image built from the PR (as long as we get the openshift/release configuration right, which will be another step), it's going to be much easier to line up the pieces if you do it in stages as i described.

Reply:
(this is part of why the `images` job is failing on this PR currently)

Reply:
I split the RAG content image part of this PR into https://github.com/openshift/lightspeed-service/pull/556.